test/e2e/page-objects/pages/network-manager.ts

```diff
@@ -142,7 +142,7 @@ class NetworkManager {
     console.log(`Checking if ${tabName} tab is selected`);
     // Find the active tab and verify it contains "Custom" text
     await this.driver.waitForSelector({
-      css: `${this.tabList} li.tab--active button`,
+      css: `${this.tabList} div.tab--active button`,
```

Comment:
Unfortunately the new Box has yet to have an `asChild` prop so we don't have the ability to change the underlying html element working on a PR here https://github.com/MetaMask/metamask-design-system/pull/834